images/runtime/php/Dockerfile.template

```diff
@@ -0,0 +1,114 @@
+# Startup script generator
+FROM golang:1.11-stretch as startupCmdGen
+WORKDIR /go/src/startupscriptgenerator/
+COPY src/startupscriptgenerator .
+ARG GIT_COMMIT=unspecified
+ARG BUILD_NUMBER=unspecified
+ENV GIT_COMMIT=${GIT_COMMIT}
+ENV BUILD_NUMBER=${BUILD_NUMBER}
+# GOPATH is set to "/go" in the base image, so no need to set it again
+RUN ./build.sh php /opt/startupcmdgen/startupcmdgen
+
+FROM php:%PHP_BASE_IMAGE%
+SHELL ["/bin/bash", "-c"]
+ENV PHP_VERSION %PHP_VERSION%
+
+# Bake Application Insights key from pipeline variable into final image
+ARG AI_KEY
+ENV ORYX_AI_INSTRUMENTATION_KEY=${AI_KEY}
+
+# COPY images/runtime/python/install-dependencies.sh /tmp/scripts/install-dependencies.sh
+# RUN /tmp/scripts/install-dependencies.sh
```

Comment:
Can you see if rearranging the content here is feasible? For example, any trivial change in this file would bust the cache from this point on and all the dependencies in line 35 (`Install common PHP extensions`) would get installed again.